03_letters/mohitnag/main_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLetters(t *testing.T) {
+	type testCases struct {
+		input    string
+		expected map[rune]int
+	}
+	testData := map[string]testCases{
+		"Happy Case": {"eaa!baabq23123!",
+			map[rune]int{33: 2, 49: 1, 50: 2, 51: 2, 97: 4, 98: 2, 101: 1, 113: 1}},
+		"Empty string should return empty map": {"", map[rune]int{}},
+		"Unicode string test":                  {"\u00C0\u00C0\u00C0\u00C0\u00C1\u00C1", map[rune]int{192: 4, 193: 2}},
```

Comment:
You can also do the same here. If you are on a mac, you can access the emoji keyboard by clicking "command, control, space" which allows you to enter all kinds of special unicode values. Use this and it will be a lot clearer.
Eg.

```go
"Unicode string test":                  {"😀😀😀😛😛😛", map[rune]int{😀: 3, 😛: 3}}
```

Reply:
Done